ee/clickhouse/models/test/test_element.py

```diff
@@ -0,0 +1,83 @@
+from ee.clickhouse.client import sync_execute
+from ee.clickhouse.models.element import create_elements, get_all_elements, get_elements_by_elements_hash
+from ee.clickhouse.util import ClickhouseTestMixin
+from posthog.api.test.base import BaseTest
+from posthog.models import Element
+
+
+class TestClickhouseElement(ClickhouseTestMixin, BaseTest):
+    def test_create_elements(self) -> None:
+        elements_hash_1 = create_elements(
+            team=self.team,
+            elements=[
+                Element(tag_name="a", href="/a-url", nth_child=1, nth_of_type=0),
+                Element(tag_name="button", nth_child=0, nth_of_type=0),
+                Element(tag_name="div", nth_child=0, nth_of_type=0),
+                Element(tag_name="div", nth_child=0, nth_of_type=0, attr_id="nested",),
+            ],
+            use_cache=False,
+        )
+
+        self.assertEqual(len(get_all_elements()), 4)
```

Comment:
We need to develop a better way to do end to end testing that is more deterministic than what we have right now. This will be tricky until then.